bento_beacon/utils/censorship.py

```diff
@@ -1,17 +1,42 @@
 from flask import current_app, g
 from .exceptions import APIException, InvalidQuery
+from .katsu_utils import katsu_censorship_settings
 
 
 MESSAGE_FOR_CENSORED_QUERY_WITH_NO_RESULTS = "No results. Either none were found, or the query produced results numbering at or below the threshold for censorship."
 
 
+def set_censorship_settings(max_filters, count_threshold):
+    current_app.config["MAX_FILTERS"] = max_filters
+    current_app.config["COUNT_THRESHOLD"] = count_threshold
+
+
+# saves settings to config as a side effect
+def censorship_retry():
```

Comment:
would be nice to have return type hints on this function and the two below (`tuple[int | None, int | None]`, `int | None`), but otherwise this PR looks good

Reply:
Thanks... I've started adding these incrementally where they're helpful